### ISSUE TITLE
ASL Popup Player — Feature Request

### ISSUE THREAD

**Issue by omenking** [vidstack/vidstack#642]
I produce free educational content.
I deliver this video content via plyr.js

I know deaf engineers and ASL interprets who are willing to volunteer to sign my free educational content however there is no means current means to deliver accessible ASL overlays. (the majority have to burn the ASL overlay into the video obscuring information)

<img width="1262" alt="Screen Shot 2022-03-30 at 9 51 52 AM" src="https://user-images.githubusercontent.com/7776/160850817-76a70296-0a87-4385-a095-8bd0c45f2536.png">

https://dcmp.org/learn/679-asl-pop-up-player

**Comment by mihar-22** [comment]
Hey @omenking thanks for sharing this pain point. I love hearing about interesting use-cases like this. This seems to be a duplicate of https://github.com/vidstack/vidstack/issues/625 but I'll leave it open since you're requesting it for a different use-case.

This will fall under the responsibility of the `<vds-media-sync>` component. I haven't worked in current time synchronization (or more broadly playback) just yet and I think it's going to be tough but not impossible.

For playback we need to figure out how to handle the following:

- Initial load (all media needs to load before we can start).
- Buffering delays (what to do if one of the media is buffering).
- Time sync (seeking and time drift). There's going to be a slight drift in playback no matter what due to CPU scheduling and all that jazz. I think the best way to tackle this is some formula that will speed some media slightly up to ensure it catches back up. Probably need some notion of master and slave as well so we know what timstamp all media needs to stay in-sync with.

These are definitely hard problems but I think it's worth giving it a solid shot, especially if it helps in terms of accessibility .

**Comment by tomByrer** [comment]
> * There's going to be a slight drift in playback no matter what due to CPU scheduling

I didn't know that... does VTT keep in sync?
